src/ScottPlot5/ScottPlot5/Plottables/OHLCPlot.cs

```diff
@@ -0,0 +1,64 @@
+﻿using ScottPlot.Axis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ScottPlot.Plottables
+{
+    public class OHLCPlot : IPlottable
+    {
+        public bool IsVisible { get; set; } = true;
+        public IAxes Axes { get; set; } = Axis.Axes.Default;
+        public readonly DataSources.IOHLCSource Data;
+        
+        public LineStyle GrowingStyle { get; } = new() { Color = Colors.LightGreen, Width = 2 };
+        public LineStyle FallingStyle { get; } = new() { Color = Colors.LightCoral, Width = 2 };
+
+        /// <summary>
+        /// The number of pixels taken up by each wick
+        /// </summary>
+        public int Width { get; set; } = 10;
+
+        public OHLCPlot(DataSources.IOHLCSource data)
+        {
+            Data = data;
+        }
+
+        public IEnumerable<LegendItem> LegendItems => Enumerable.Empty<LegendItem>();
+
+        public AxisLimits GetAxisLimits() => Data.GetLimits();
+
+        public void Render(SKSurface surface)
+        {
+            using SKPaint paint = new();
+            using SKPath path = new();
+
+            foreach ((double x, OHLC y) in Data.GetOHLCPoints())
+            {
```

Comment:
If we are concerned about performance, then it would be more efficient to have 2 paths: Growing and Falling.
Accumulate them in a loop.
Then, after the loop, call for drawing 1 time each Path.

Reply:
I don't think it's necessary (or even noticeable) on performance grounds, but I do think the resulting code is simpler (no pesky `path.Reset()` to remember) so I implemented it anyway. It was something I thought of about 2 seconds after I put this PR down for the day -_-

Reply:
> performance

I agree that performance is probably not an issue for OHLC plots. 

Only fit a few hundred OHLC symbols can fit on a plot before they become too small and overlapping to be interpretable. Maybe in the future some sort of "high density mode" could be devised, but for now I'm satisfied with whatever is simplest and not optimizing financial plots for performance